packages/desktopjs-openfin/src/openfin.ts

```diff
@@ -225,10 +223,10 @@ export class OpenFinContainerWindow extends ContainerWindow {
  * @augments MessageBus
  */
 export class OpenFinMessageBus implements MessageBus {
-    private bus: fin.OpenFinInterApplicationBus;
+    private bus: any;
     private uuid: string;
 
-    public constructor(bus: fin.OpenFinInterApplicationBus, uuid: string) {
+    public constructor(bus: any, uuid: string) {
```

Comment:
Avoid exposing OpenFin types to alleviate pain of upstream consumer.

Reply:
same as above, we should add this to release notes

Reply:
Will be in release notes